src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -3575,6 +3575,10 @@
           "type": "string",
           "description": "The unique identifier of the expected expense"
         },
+        "isPaymentCardRelated": {
+          "type": "boolean",
+          "description": "Indicates if the expected expense entry has the `Is Payment Card Related` field enabled"
```

Comment:
I'm seeing two different quotation marks here `Is Payment Card Related` and ‘sendback’ 
are you sure the text is displayed as expected

Reply:
fixed